Mathlib/Analysis/SpecialFunctions/Trigonometric/Bounds.lean

```diff
@@ -88,25 +88,25 @@ lemma sin_le_mul (hx : -(π / 2) ≤ x) (hx₀ : x ≤ 0) : sin x ≤ 2 / π * x
 /-- Half of **Jordan's inequality** for absolute values. -/
 lemma mul_abs_le_abs_sin (hx : |x| ≤ π / 2) : 2 / π * |x| ≤ |sin x| := by
   wlog hx₀ : 0 ≤ x
-  case inr => simpa using this (by rwa [abs_neg]) <| neg_nonneg.2 <| le_of_not_le hx₀
+  · simpa using this (by rwa [abs_neg]) <| neg_nonneg.2 <| le_of_not_le hx₀
   rw [abs_of_nonneg hx₀] at hx ⊢
   exact (mul_le_sin hx₀ hx).trans (le_abs_self _)
 
 lemma sin_sq_lt_sq (hx : x ≠ 0) : sin x ^ 2 < x ^ 2 := by
   wlog hx₀ : 0 < x
   case inr =>
     simpa using this (neg_ne_zero.2 hx) <| neg_pos_of_neg <| hx.lt_of_le <| le_of_not_lt hx₀
-  rcases le_or_lt x 1 with hxπ | hxπ
```

Comment:
What's the justification for this change? `rcases`, unlike `cases'`, is encouraged syntax.

Reply:
I'm not sure if the case names from `rcases` are stable; in any case there's no advantage to using `rcases` here, and separating the case names from the names of the introduced hypotheses is less readable, IMO.

Reply:
The case names from `rcases` correspond exactly with the case names from `cases`, so this change doesn't affect stability in that sense. There's also no advantage to using `cases` here, the introduced hypotheses aren't referred to by name in the proof.

And in later cases, the `rfl` syntax for `rcases` makes the proof clearer and simpler to read, so there is a stronger advantage which is not allowed with the `cases` proof.

Reply:
I've dropped the case-related changes from this PR